mantidproject/mantid
# Export WorkspaceWidget to workbench

Exported the WorkspaceWidget to the workbench. Key features of this PR:
* Addition of MantidTreeModel (basic implementation on MantidUI - without the MantidPlot specifics)
* SIP wrapping for WorkspaceWidget
* Testcases and additions into workbench of SIP wrapper

**To test:**

* Code review

* Check the WorkspaceWidget runs as expected in the in the workbench
  * The widget appears in the bottom left
  * All the buttons at the top of the widget work as expected (same as MantidPlot)
  * All the *available* options from the context menu (right click on a workspace) work as expected.

Fixes #21442

**Release Notes** 
*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
